zerberus/voice.cpp

```diff
@@ -120,8 +121,12 @@ void Voice::start(Channel* c, int key, int v, const Zone* z)
       if (trigger == Trigger::CC)
             _velocity = 127;
       float curve = _velocity * _velocity / (127.0 * 127.0);
+
+      double rt_decay_value = 1.0;
+      if (trigger == Trigger::RELEASE)
+            rt_decay_value = pow(10, (-z->rtDecay * durSinceNoteOn)/20);
```

Comment:
0.005 was there from the beginning - not sure where this comes from - I think just something to have a little more headroom. Maybe ask Werner if he knows something? The 20 comes from decibel https://en.wikipedia.org/wiki/Decibel#Field_quantities_and_root-power_quantities